src/main/kotlin/no/nav/amt/deltaker/bff/deltaker/DeltakerService.kt

```diff
@@ -99,9 +106,65 @@ class DeltakerService(
                 godkjent = LocalDateTime.now(),
                 gyldigTil = null,
                 deltakerVedSamtykke = deltaker,
-                godkjentAvNav = forslag.godkjentAvNav,
+                godkjentAvNav = oppdatertDeltaker.godkjentAvNav,
+            ),
+        )
+    }
+
+    fun oppdaterDeltaker(
```

Comment:
Skal vi håndtere på noen måte at samme informasjonen kommer inn to ganger her? feks om bakgrunnsinformasjon blir sendt inn to ganger (samme tekst bare annen "endretAv") så vil endretAv ikke være "riktig"?

Reply:
ja, det var lurt, hvis det er nøyaktig samme endring kan vi forkaste den :)